packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts

```diff
@@ -331,11 +324,38 @@ export class AzureCommunicationChatAdapter implements ChatAdapter {
 
   /* @conditional-compile-remove(teams-inline-images) */
   async downloadAuthenticatedAttachment(attachmentUrl: string): Promise<string> {
-    if (!this.fileUploadAdapter.downloadAuthenticatedAttachment) {
-      return '';
-    }
+    return this.asyncTeeErrorToEventEmitter(async () => {
```

Comment:
@PorterNan We chatted about moving this out of the file upload adapter, perhaps it would be a good idea to move this into a new `DownloadUtils` instead?